code/ATMOSPHERICS/components/unary_devices/tank.dm

```diff
@@ -66,5 +44,6 @@
 /obj/machinery/atmospherics/components/unary/tank/air/New()
 	..()
 	var/datum/gas_mixture/air_contents = AIR1
-	air_contents.oxygen = AIR_CONTENTS * 0.2
-	air_contents.nitrogen = AIR_CONTENTS * 0.8
\ No newline at end of file
+	air_contents.assert_gases("o2", "n2")
+	air_contents.gases["o2"][MOLES] = AIR_CONTENTS * 0.2
```

Comment:
Accessing the map (associative list) should be done with defines, as a small misspelling would be very annoying and a pain to debug. A define would let you know if you did not spell it correctly, without having to test it in runtime.

Not to mention it is just good practice, especially for string-key-based maps.